Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Components/Sidebar.swift

```diff
@@ -81,6 +81,24 @@ struct CommunitySidebarView: View {
                         .foregroundColor(.secondary)
                 }
             }
+            
+            Section("Server")
+            {
+                if let communityServer = communityDetails.communityView.community.actorId.host() {
+                    if let communityProtocol = communityDetails.communityView.community.actorId.scheme {
+                        Link(communityServer, destination: URL(string: "\(communityProtocol)://\(communityServer)")!)
```

Comment:
given the `.actorId` is already a non-optional `URL` could we not simplify this to:
```swift
if let communityServer = communityDetails.communityView.community.actorId.host() {
    Link(communityServer, destination: communityDetails.communityView.community.actorId)
} else {
    Text("n/a")
        .foregroundColor(.secondary)
}
```

it'd save re-constructing the original URL we already have on hand?

Reply:
That will result in a link to the community instead of the main server its self but that is definitely simpler.

Reply:
heh, simpler _and_ wrong 🎉 

apologies, didn't realise we were needing to strip the path component from it - annoying there's not a standard library method that gives us that 😞